src/gtfs-rt/client.js

```diff
@@ -0,0 +1,72 @@
+import GtfsRealtimeBindings from 'gtfs-realtime-bindings';
+import logger from '../lib/logger.js';
+
+const API_KEY = process.env.GTFS_REALTIME_TOKEN;
```

Comment:
can you move these `process.env` refs into the src/config.js file, then import that here? My goal with that file was to keep all env var refs confined to there for simply reference and to make the eventual shift to dependency injection easier.

Reply:
done